### PR TITLE
Add some initial info on how to apply

### PR DESCRIPTION
**Pull request by Sing-Li**
.. now that we have students actively looking

### CODE REVIEW

File: 1. Contributing/Google Summer of Code/README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Google Summer of Code 2017
 
+## How to apply
+
+Interested students should email the mentor(s) directly for discussion and further information.  You can also find them, most of the time, on our community server https://demo.rocket.chat/  for real-time chat.   Many of the mentors are fluent in both Portugeuese and English.  If you have ideas and proposals that is not on our idea list, or if a mentor is not available, please email to gsoc17@rocket.chat
```
Comment (graywolf336) [inline 1. Contributing/Google Summer of Code/README.md]: `contact+gson17@rocket.chat` is what the website now lists

**Review by graywolf336 — DISMISSED** [review]

File: 1. Contributing/Google Summer of Code/README.md (hunk 1 above)
Comment (marceloschmidt) [reply 1. Contributing/Google Summer of Code/README.md]: portuguese*

Comment (geekgonecrazy) [reply 1. Contributing/Google Summer of Code/README.md]: Yeah we'll need to hold off until we get that alias created or use `contact+gsoc17@rocket.chat`

Comment (Sing-Li) [reply 1. Contributing/Google Summer of Code/README.md]: is it contact+gson17 or contact+gsoc17  ?   changing for latter

**Review by Sing-Li — COMMENTED** [review]
thx

**Review by geekgonecrazy — APPROVED** [review]

**Pull request merged**